modules/openapi/src/internals/postprocess/ExternalMemberRefTransformer.scala

```diff
@@ -0,0 +1,106 @@
+/* Copyright 2022 Disney Streaming
+ *
+ * Licensed under the Tomorrow Open Source Technology License, Version 1.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://disneystreaming.github.io/TOST-1.0.txt
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package smithytranslate.openapi.internals
+package postprocess
+
+import org.typelevel.ci._
+import cats.data.NonEmptyChain
+import cats.data.Chain
+import scala.annotation.tailrec
+
+/** This transformer is for the specific case where an external reference
```

Comment:
as I recall from our discussion, this only happens if B$b is a primitive right?
can we state that in the doc or we don't want to because it may happen in other instances and we're not sure

Reply:
Yeah I will add it to the doc. It is that way since we match on the NewType on line 95 below and at this point in the postprocessing, all primitives are wrapped in newtypes